docs/usage/shoot_settings/default_seccomp_profile.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Default Seccomp Profile
+description: Enable the use of `RuntimeDefault` as the default seccomp profile through `spec.kubernetes.kubelet.seccompDefault`
+---
+
+# Default Seccomp Profile and Configuration 
+
+This is a short guide describing how to enable the defaulting of seccomp profiles for Gardener managed workloads in the seed. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in this [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/security/seccomp/).
```

Comment:
```suggestion
This is a short guide describing how to enable the defaulting of seccomp profiles for Gardener managed workloads for a Shoot cluster. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in this [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/security/seccomp/).
```